index.html

```diff
@@ -16,6 +16,169 @@
+          nisl tempus, quis finibus diam porta. Aliquam egestas id dolor dictum
+          ultricies. Curabitur quis lorem eleifend, laoreet elit non, blandit
+          odio. Cras diam ex, egestas ac tempus in, bibendum a dui. Suspendisse
+          potenti. Nulla in ex id eros maximus porta. In lorem velit, efficitur
+          accumsan sem ut, ultricies finibus metus. Sed cursus imperdiet orci
+          sodales congue.
+        </p>
+        <img
+          src="https://images.unsplash.com/photo-1671726805768-575f88de945a?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80"
+          alt="a laptop on a busy desk denoting software being used"
+        />
+        <p>
+          Phasellus non ex eu orci consequat commodo. Curabitur at urna magna.
+          Curabitur ultrices ultrices nulla eget gravida. Aenean id sapien
+          mauris. Sed ut arcu orci. Integer interdum tellus et lorem sodales
+          eleifend. Mauris fermentum non sem molestie bibendum. Nulla non
+          condimentum orci, et pellentesque sapien. Duis egestas risus et
+          fermentum egestas. Donec dignissim iaculis placerat. Etiam tempus, dui
+          eget fermentum tristique, risus mi euismod est, in molestie ligula
+          felis non erat. Praesent imperdiet dolor ut libero gravida, vitae
+          euismod nulla vehicula. Integer eget sem id diam eleifend pretium ac a
+          lectus. Maecenas vestibulum interdum lobortis. Integer id laoreet
+          nibh, eget gravida erat.
+        </p>
+        <p>
+          Below you will find a link to a creative called PixImperfect on
+          Youtube:
+        </p>
+        <a href="https://www.youtube.com/@PiXimperfect"
+          >Click Here for PixImperfect</a
+        >
+      </article>
+      <article>
+        <h2>Artificial Intelligence in Photography</h2>
+        <p>
+          AI in photography and design has really taken off with a lot of
+          interest from "non-technies" who are using Apps such as Lensa AI.
+        </p>
+        <p>
+          An excellent launch point is Mid-Journey which is a prompt based image
+          generation service.
+        </p>
+        <p>
+          Integer ultrices placerat iaculis. In accumsan, risus sed cursus
+          consectetur, velit risus consectetur diam, vitae posuere nulla mi sit
+          amet nisi. Maecenas finibus ultrices hendrerit. Pellentesque volutpat
+          nulla gravida laoreet efficitur. Donec mattis urna quis erat
+          fermentum, ac pulvinar risus vulputate. Cras pharetra viverra lorem
+          sit amet vehicula. Pellentesque consectetur lobortis enim eget
+          lobortis. Aliquam erat volutpat. Phasellus sit amet viverra neque, a
+          cursus diam.
+        </p>
+        <img
+        src="https://images.unsplash.com/photo-1659468480453-f04eb04661cb?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1864&q=80
```

Comment:
```suggestion
        src="https://images.unsplash.com/photo-1659468480453-f04eb04661cb?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1864&q=80"
```

Looks like this is missing an end quote. Please make sure you have Prettier installed and checking your syntax. https://syllabus.codeyourfuture.io/guides/code-style-guide